README.md

```diff
@@ -43,6 +43,13 @@ platforms](#setup-on-other-platforms).
    This will automatically install both the `lorri` command and `direnv`
    (both required for the next steps).
 
+   **Note**: There's [a known issue](https://github.com/target/lorri/issues/374
+   "nixpkgs `services.lorri` requires reboot for the user lorri.socket to appear")
+   preventing the lorri daemon from starting automatically upon installation. 
+   Until it's resolved, you'll have to reboot, or reload the user daemon by hand
+   by running `systemctl --user daemon-reload`.
```

Comment:
```suggestion
   Until it's resolved, you'll have to reload the user daemon by hand by
   running `systemctl --user daemon-reload`, or reboot.
```

I think that running `systemctl --user daemon-reload` is preferable to rebooting, so I suggest we mention this option first.